Vinyl/Vinyl.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-struct Vinyl {
+public struct Vinyl {
     
     let tracks: [Track]
```

Comment:
Since `Vinyl` is public and you can create vinyls manually maybe it would be helpful expose this publicly.